### DISCUSSION TITLE
demo的parse_text方法存在瑕疵

### DISCUSSION THREAD

**Discussion by rongdi** [zai-org/ChatGLM3#506] [Bad Case]
![image](https://github.com/THUDM/ChatGLM3/assets/5162800/d2af8e23-77d1-4aac-9d72-d182e62eed22)
使用stream_chat时这里的line刚出现"```"时，会被替换成<pre><code class="language-">，这时会没有语言，如下
![image](https://github.com/THUDM/ChatGLM3/assets/5162800/3444f7c1-bd61-4aa4-ab6a-b14cc5f58047)
可以加上如下条件
![image](https://github.com/THUDM/ChatGLM3/assets/5162800/3158b4d9-3cd6-444d-bd56-6038fd907ed9)

**Comment by zRzRzRzRzRzRzR** [comment]
有没有好的复现方法，我去复现一个
